libs/openFrameworks/graphics/ofBitmapFont.h

```diff
@@ -22,7 +23,7 @@ class ofBitmapFont{
 	~ofBitmapFont();
 	ofMesh getMesh(const string & text, int x, int y, ofDrawBitmapMode mode=OF_BITMAPMODE_MODEL_BILLBOARD, bool vFlipped=true) const;
 	const ofTexture & getTexture() const;
-	ofRectangle getBoundingBox(const string & text, int x, int y) const;
+	ofRectangle getBoundingBox(const string & text, int x, int y, ofDrawBitmapMode mode = ofGetStyle().drawBitmapMode, bool vFlipped = ofIsVFlipped()) const;
```

Comment:
`ofGetStyle().drawBitmapMode` looks funny, but I'm assuming it's the only way to do this to preserve backward compatibility right?